src/modules/dashboard/sagas/actions/manageReputation.ts

```diff
@@ -71,16 +68,18 @@ function* smiteAction({
         },
       });
 
-    yield createGroupTransaction(emitDomainReputationPenalty, {
+    yield createGroupTransaction(manageReputation, {
       context: ClientType.ColonyClient,
-      methodName: 'emitDomainReputationPenaltyWithProofs',
+      methodName: isSmitingReputation
+        ? 'emitDomainReputationPenaltyWithProofs'
+        : 'emitDomainReputationReward',
```

Comment:
Why doesn't reward require proofs when sending. I think it should.

Please check with Alex and then let me know, as if it does, I need to build it into `colonyJS`

Reply:
In the words of our local wizard, Alex:

> Reward requires the root permission, which you do not need to provide a proof when you use
Penalty requires the arbitration permission, which does require a proof to leverage

With that being said, I thought rewarding required the arbitration permission, so I need to do some changes in my award dialog PR to reflect that it's the root permission the one that is required

Reply:
Great to hear then. Less work for me :P